docs/upgrade-guide-1.6-to-1.7.ipynb

```diff
@@ -29,31 +41,34 @@
     ]
    },
    "source": [
-    "## Prerequisites\n",
+    "## Before upgrade\n",
+    "\n",
+    "**WARNING: To prevent catastrophic data loss all important data should be backed up according to the policies of your organisation.**\n",
```

Comment:
I know the message is to convey severity here, but I think just "data loss" comes across with enough.  Just a suggestion

```suggestion
    "**WARNING: To prevent data loss, all important data should be backed up according to the policies of your organisation.**\n",
```